react/src/lib/List/index.js

```diff
@@ -422,6 +423,8 @@ List.propTypes = {
   itemRole: PropTypes.string,
   /** @prop Callback function invoked by user selecting an interactive item within List | null */
   onSelect: PropTypes.func,
+  /** @prop Disables the stopPropagation() & preventDefault() for ArrowKey Events | false */  
+  propogateKeyDownEvents: PropTypes.bool,
```

Comment:
Let's call this shouldPropagateKeyDown...to match the other booleans of this component.

Reply:
Done.